detection_rules/rule.py

```diff
@@ -1123,10 +1123,19 @@ def _post_dict_conversion(self, obj: dict) -> dict:
     def to_api_format(self, include_version: bool = True) -> dict:
         """Convert the rule to the API format."""
 
+
+    def get_hashable_content(self, include_version: bool = False, include_integrations: bool = True) -> dict:
```

Comment:
When we ship this, all integrations will invariably get a version bump because the sha256 would change with dropping the related_integrations and the first time all of the integrations rules will see a bump.